src/admin-views/tabbed-view/tabbed-view.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * The default template for a Tabbed View.
+ *
+ * @var Tribe__Tabbed_View $view
+ */
+
+/** @var Tribe__Tabbed_View__Tab[] $tabs */
+$tabs = $view->get_visibles();
+?>
+
+<?php if ( count( $tabs ) > 1 ) : ?>
+    <div class="tabbed-view-wrap wrap">
+		<?php if ( $view->get_label() ) : ?>
```

Comment:
The formatting/indentation here seems slightly odd to me (ditto for the `foreach` that follows) - shouldn't the control statements be tabbed back a bit and the HTML inside the blocks be tabbed in?